softperipheral/doc/sQSPI/nrf54H20_porting_v0_1_0.rst

```diff
@@ -0,0 +1,246 @@
+        - Output
+        - Connected
+        - Pull-up
+      * - **P7.2**
+        - IO1
+        - Output
+        - Connected
+        - Pull-up
+      * - **P7.3**
+        - IO2
+        - Output
+        - Connected
+        - Pull-up
+      * - **P7.4**
+        - IO3
+        - Output
+        - Connected
+        - Pull-up
+      * - **P7.5**
+        - CSN
+        - Output
+        - Disconnected
+        - No Pull
+
+   .. tab:: **P9** GPIO pins
+
+    .. list-table::
+      :widths: 10 10 20 20 20
+      :header-rows: 1
+
+      * - GPIO pin
+        - sQSPI Role
+        - Direction config
+        - Input config
+        - Pin Pull config
+      * - **P9.0**
+        - SCK
+        - Output
+        - Disconnected
+        - No Pull
+      * - **P9.1**
+        - IO0
+        - Output
+        - Connected
+        - Pull-up
+      * - **P9.2**
+        - IO1
+        - Output
+        - Connected
+        - Pull-up
+      * - **P9.3**
+        - IO2
+        - Output
+        - Connected
+        - Pull-up
+      * - **P9.4**
+        - IO3
+        - Output
+        - Connected
+        - Pull-up
+      * - **P9.5**
+        - CSN
+        - Output
+        - Disconnected
+        - No Pull
+
+.. note::
+   sQSPI driver provides a default GPIO configuration and multiplexing.
+   This setup is applied when both :c:var:`nrfx_qspi2_cfg_t.skip_gpio_cfg` and :c:var:`nrfx_qspi2_cfg_t.skip_pmux_cfg` are set to ``false``.
+
+Configuring pins
+================
+
+In some cases you might have to modify the sQSPI driver configuration.
+For example, when changing pin drive strength to guarantee signal integrity for a new PCB design.
+You must address these cases on the sQSPI application code:
+
+* If you set the :c:var:`nrfx_qspi2_cfg_t.skip_gpio_cfg` variable to ``true``, the GPIO configuration is not managed by the sQSPI driver and it must be manually handled by the application. 
+  This is a requirement for the nRF54H20 device.
+* If you set the :c:var:`nrfx_qspi2_cfg_t.skip_pmux_cfg` variable to ``true``, the GPIO multiplexing is not managed by the sQSPI driver and it must be manually handled by the application. 
+  This is a requirement for the nRF54H20 device.
+
+GPIO multiplexing must be handled by setting the correct ``CTRLSEL`` value in UICR.
+
+Additionally, the sQSPI application requires FastPad bias configuration to run at high frequency on SCLK.
+
+Memory retention configuration
+******************************
+
+The sQSPI Soft Peripheral requires RAM retention in order to go into the lowest power consumption mode, which can be called through the :c:func:`nrfx_qspi2_deactivate` API.
+
+.. _nrf54H20_porting_guide_ram_configuration:
+
+RAM Configuration
+*****************
+
+The sQSPI Soft Peripheral operates from RAM.
+Your build environment must reserve the required RAM and ensure that it is readable and writable by both the application core and the FLPR core.
+This table details the memory region, it should be non-cacheable:
```

Comment:
nit:
```suggestion
This table details the memory region, it must be non-cacheable:
```
If we use softer wording like "shall" or "should", we should describe what happens if the reader decides not to follow the recommendation.